app/models/work.rb

```diff
@@ -9,16 +9,13 @@ class Work < ApplicationRecord
   has_many :related_works, dependent: :destroy
   has_many_attached :files
 
-  validates :abstract, presence: true
-  validates :access, presence: true
-  validates :citation, presence: true
-  validates :contact_email, presence: true
-  validates :created_edtf, presence: true
-  validates :license, presence: true
-  validates :state, presence: true
-  validates :subtype, presence: true
-  validates :title, presence: true
-  validates :work_type, presence: true
+  validates :abstract, :access, :citation, :contact_email, :created_edtf,
+            :license, :state, :subtype, :title, :work_type, presence: true
```

Comment:
ya know, I wondered if I could do this ...  glad to know I could've.

Reply:
:man_dancing: